### PR TITLE
Android O

### PR DESCRIPTION
**Pull request by danielpassos**
Related issues:

* [FH-4344](https://issues.jboss.org/browse/FH-4344) - Update oauth-android-app to Android O

### CODE REVIEW

File: app/src/main/java/com/feedhenry/oauth/FHOAuth.java [hunk 1]
```diff
@@ -13,25 +13,22 @@
...
+import com.feedhenry.sdk.api.*;
 
 /**
  * This abstract class contains all of the important FHOAuth code.
  */
 public abstract class FHOAuth extends AppCompatActivity {
 
     private static final String TAG = "FHAuthActivity";
-    private static final String FH_AUTH_POLICY = "Google";  //"Google" should be replaced with policy id you created;
+    private static final String FH_AUTH_POLICY = "Google_Android";  //"Google" should be replaced with policy id you created;
```
Comment (jcesarmobile) [inline app/src/main/java/com/feedhenry/oauth/FHOAuth.java]: README and the comment say "Google", so don't change it

File: circle.yml [hunk 2]
```diff
@@ -22,13 +22,5 @@ dependencies:
...
-    # Launch the emulator
-    - emulator -avd $ANDROID_EMULATOR -no-window:
-        background: true
-        parallel: true
-    # Wait to emulator get ready
-    - circle-android wait-for-boot
-
   override:
-    - ./gradlew build connectedCheck
+    - ./gradlew build build
```
Comment (jcesarmobile) [inline circle.yml]: is `build build` right? I think one `build` is enough

Comment (danielpassos) [reply circle.yml]: Fixed!

**Comment by secondsun** [comment]
So IIRC OAuth is still broken in Studio so there isn't a way to test this without adding an intent redirect into fh-android-sdk.  ThoughtS?

**Comment by damienomurchu** [comment]
Visually the changes look good, but have not tested yet due to ouath studio issue mentioned by @secondsun 

1 point - given we've deleted the tests, are the following files needed anymore?
- app/src/androidTest/AndroidManifest.xml
- app/src/androidTest/assets/fhconfig.local.properties
- app/src/androidTest/assets/fhconfig.properties

**Comment by danielpassos** [comment]
@damienomurchu Nice catch. Removed!!!

**Comment by danielpassos** [comment]
@jcesarmobile => @johnfriz said let's leave it like it for now.

**Comment by jcesarmobile** [comment]
Ok, so I say, merge this even if we can't test if it works. It loads the Google website saying that that user agent is not allowed, so "good" enough to me.

**Pull request merged**